funcs/func_respoke_endpoint.c

```diff
@@ -40,7 +40,9 @@
 
 #include "asterisk.h"
 
+#if ASTERISK_MAJOR_VERSION >= 13 && ASTERISK_MAJOR_VERSION < 15
```

Comment:
Are `AST_VERSION_MAJOR` and this variable `ASTERISK_VERSION_MAJOR` deliberately different?